src/InstallationDirectories.cc

```diff
@@ -28,27 +28,28 @@ namespace gz
 {
 namespace physics
 {
-inline namespace GZ_PHYSICS_VERSION_NAMESPACE {
```

Comment:
given that this is still in the header file, is it ok to change it here?

https://github.com/gazebosim/gz-physics/blob/mjcarroll/garden_bazel/include/gz/physics/InstallationDirectories.hh#L30

Reply:
I just moved it a little further down (https://github.com/gazebosim/gz-physics/blob/ddcd72b79b6341bf69d6f27da97ccca7b840a10f/src/InstallationDirectories.cc#L149), and put the rest in an anonymous namespace since it's not part of the public API

Reply:
👍